app/src/main/java/com/keylesspalace/tusky/components/notifications/NotificationFetcher.kt

```diff
@@ -65,23 +66,24 @@ class NotificationFetcher @Inject constructor(
                         }
                     }
 
+                    val notificationsGrouped = groupByGroupId(account, notifications)
```

Comment:
Can you use https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/group-by.html here?

Reply:
Ok

It's amazing how many helper functions Kotlin has.
Even if it hadn't it would have made sense to write our own extension for this - as this reads much nicer and more concise now...